core/controllers/editor_test.py

```diff
@@ -1014,6 +1015,68 @@ def test_exploration_rights_handler(self):
         self.logout()
 
 
+class UserExplorationEmailsIntegrationTest(BaseEditorControllerTest):
+    """Test the handler for user email notification preferences."""
+
+    def test_user_exploration_emails_handler(self):
+        """Test user exploration emails handler."""
+
+        # Owner creates exploration
+        self.login(self.OWNER_EMAIL)
+        exp_id = 'eid'
+        self.save_new_valid_exploration(
+            exp_id, self.owner_id, title='Title for emails handler test!',
+            category='Category')
+
+        exploration = exp_services.get_exploration_by_id(exp_id)
+
+        response = self.testapp.get(
+            '%s/%s' % (feconf.EDITOR_URL_PREFIX, exp_id))
+        csrf_token = self.get_csrf_token_from_response(response)
+
+        exp_email_preferences = (
+            user_services.get_email_preferences_for_exploration(self.owner_id,
```

Comment:
FYI -- generally better to break after the '(', otherwise everything gets squeezed to the right. Optional for this PR, but please note this for future PRs.